common/transport/amqp/src/amqp.ts

```diff
@@ -11,6 +11,7 @@ import { ClaimsBasedSecurityAgent } from './amqp_cbs';
 import { SenderLink } from './sender_link';
 import { ReceiverLink } from './receiver_link';
 import { AmqpLink } from './amqp_link_interface';
+import { _ } from 'lodash';
```

Comment:
>lodash [](start = 19, length = 6)

i think you can import only a subset of the package (lodash/something) to save on load times